### PR TITLE
fix(client): for a11y, add role "application" to input monaco editor

### LINKED ISSUE #36153: [a11y] Please give the code editor a role on the page
**Describe the bug**

Based on #34222 , bullet 1:

Please give the code editor an element with a role on the page. I would suggest an input role of some kind or a role="application". Do whatever looks good, but there is currently no editor actually on the page for screen reader users and the only way we can get to it is by tabbing and most screen reader users use the arrow keys rather than tabbing when browsing webpages like this.

**To Reproduce**
1. Go to any lesson
2. Open the dev tools
3. Inspect the editor
4. Observe the `role` attribute

**Expected behavior**
`role="application"`

**Desktop (please complete the following information):**
 - OS: All
 - Browser All
 - Version All

**Additional context**

### PR DESCRIPTION
**Pull request by xnt**
- [X] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `master` branch of freeCodeCamp.
- [X] None of my changes are plagiarized from another source without proper attribution.
- [X] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [X] My changes do not use shortened URLs or affiliate links.

Screen Shot 2019-05-31 at 6 03 20 PM

Closes #36153

### CODE REVIEW

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
Thanks for the PR!  
Regrettably the role prop is ignored by `MonacoEditor`, so the best bet is probably just to wrap the editor in a div with that role.

File: client/src/templates/Challenges/classic/Editor.js [hunk 1]
```diff
@@ -127,6 +127,7 @@ class Editor extends Component {
           language={modeMap[ext]}
           onChange={this.onChange}
           options={this.options}
+          role={'application'}
```
[the remark this answers is not in the document]
Comment (xnt) [reply client/src/templates/Challenges/classic/Editor.js]: Awwww, bummer! Didn't look into it. I'll find a workaround when I have a downtime, thanks!

[the remark this answers is not in the document]
Comment (ojeytonwilliams) [reply client/src/templates/Challenges/classic/Editor.js]: Thanks.  Good luck!

**Comment by raisedadead** [comment]
Please feel free to open a fresh pull-request when ready.

**Pull request closed without merging**